api/tests/core/bookings/test_models.py

```diff
@@ -109,6 +113,131 @@ def test_too_many_bookings_postgresql_exception_not_executed():
     assert booking2.quantity == 2
 
 
+class BookingPopUpReactionTest:
+    @pytest.mark.parametrize(
+        "subcategory_id, pop_up_enabled",
+        [
+            (subcategories.SEANCE_CINE.id, True),
+            (subcategories.LIVRE_PAPIER.id, True),
+            (subcategories.SUPPORT_PHYSIQUE_MUSIQUE_CD.id, True),
+            (subcategories.SUPPORT_PHYSIQUE_MUSIQUE_VINYLE.id, True),
```

Comment:
Détail : si il y a True partout, pas besoin de l'ajouter au parametrize
(low [logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/))